aria/workflow_engine/tasks.py

```diff
@@ -128,16 +128,18 @@ class StartWorkflowTask(EngineBaseTask):
 
     def __init__(self, *args, **kwargs):
         super(StartWorkflowTask, self).__init__(*args, **kwargs)
-        self.on_success_signal.connect(self._create_execution_in_starage)
+        self.on_success_signal.connect(self._create_execution_in_storage)
 
     def complete(self):
-        execution = self.context.storage.execution.get(self.context.execution_id)
```

Comment:
return this execution model work
add creation of execution model in StartWorkflowTask